etc/docker/php/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env sh
+chown -R www-data:www-data -R app/cache app/logs
```

Comment:
I don't have any idea of docker, but work a little bit with Vagrant. I bet you share folder somehow between docker instance and your machine. Won't it be worthy, to move `app/cache` and `app/logs` folder somewhere outside this folder synchronization? On Vagrant even NFS have problems with i/o operations, so it resolves many problems with it.